client/webpack.config.js

```diff
@@ -212,6 +212,7 @@ const webpackConfig = {
 					},
 				},
 				prelude: `@import '${ path.join( __dirname, 'assets/stylesheets/shared/_utils.scss' ) }';`,
+				cacheDirectory: path.resolve( cachePath, 'css-loader' ),
```

Comment:
For consistency, do you think it might be nice to use the same home directory cache path as established in #44696? ([link](https://github.com/Automattic/wp-calypso/pull/44696/files#diff-c8e261d390d225bb28675f5e7d81608aR25))

This would make it much easier for devs to blow away the cache locally if necessary. Also makes it much more portable/easier when wiring this up in CI.

Reply:
I'm moving all caches under `./cache` in #44863, which I think has similar benefits.

I'm a bit concerned about moving it outside the checkout dir (i.e. to $HOME) because I'm not sure what would happen if someone has two copies of `wp-calypso` and running `yarn start` (or any other command) at the same time.

Do you think there is a big difference between `./cache` and `$HOME/.cache` for CI?

Reply:
For local development, I'd strongly prefer having everything under the `./cache` directory, inside the checked out Git folder. Quite often, when folks find their Calypso directory broken beyond repair, they can simply `rm -rf` it and checkout from scratch. Having the cache in the `$HOME/.cache` directory makes that less likely to succeed.

In CI, the benefit could be that the various "workspaces" (as CircleCI calls them) are kept separate -- the Git checkout directory, caches that are saved and restored...

Making the cache dir configurable, with a `./cache` default and overridable with an env variable in CI could be a good win-win solution.

Reply:
The problem with making it configurable is that we can't guarantee that all places using `.cache` can use that override. For example, this `./cache` will also be used for `tsc` too (https://github.com/Automattic/wp-calypso/pull/44863/files#diff-5223633ada975fe8f2ed7837c6bae13e). There may be other places in the future.